shaptools/hana.py

```diff
@@ -34,6 +35,10 @@ class HanaError(Exception):
     Error during HANA command execution
     """
 
+class FileDoesNotExist(Exception):
```

Comment:
Usually (by convention) the exception are name with `Error` sufix: `FileDoesNotExistError`